pytest/tests/sanity/rpc_state_changes.py

```diff
@@ -17,407 +17,556 @@
 
 nodes = start_cluster(4, 0, 1, None, [["epoch_length", 10], ["block_producer_kickout_threshold", 80]], {})
 
-# Plan:
-# 1. Create a new account.
-# 2. Observe the changes in the block where the receipt lands.
-
-status = nodes[0].get_status()
-latest_block_hash = status['sync_info']['latest_block_hash']
-new_account_id = 'rpc_key_value_changes'
-create_account_tx = transaction.sign_create_account_tx(
-    nodes[0].signer_key,
-    new_account_id,
-    5,
-    base58.b58decode(latest_block_hash.encode('utf8'))
-)
-new_account_response = nodes[0].send_tx_and_wait(create_account_tx, 10)
-
-state_changes_in_block_request = {
-    "block_id": new_account_response['result']['receipts_outcome'][0]['block_hash'],
-}
-
-expected_changes_in_block_response = {
-    "block_hash": state_changes_in_block_request['block_id'],
-    "changes": [
-        {
-            "type": "account_touched",
-            "account_id": new_account_id,
+def assert_changes_in_block_response(request, expected_response):
+    for node_index, node in enumerate(nodes):
+        response = node.get_changes_in_block(request)
+        assert 'result' in response, "the request did not succeed: %r" % response
+        response = response['result']
+        diff = deepdiff.DeepDiff(expected_response, response)
+        assert not diff, \
+            "query node #%d same changes gives different results %r (expected VS actual):\n%r\n%r" \
+            % (node_index, diff, expected_response, response)
+
+
+def assert_changes_response(request, expected_response, **kwargs):
+    for node_index, node in enumerate(nodes):
+        response = node.get_changes(request)
+        assert 'result' in response, "the request did not succeed: %r" % response
+        response = response['result']
+        diff = deepdiff.DeepDiff(expected_response, response, **kwargs)
+        assert not diff, \
+            "query node #%d same changes gives different results %r (expected VS actual):\n%r\n%r" \
+            % (node_index, diff, expected_response, response)
+
+
+def test_changes_with_new_account():
```

Comment:
@SkidanovAlex @bowenwang1996 @ailisp BTW, the benefit of having the tests in functions it that you can run the test suite with `pytest` (Python testing framework) and have a nice output for free:

![image](https://user-images.githubusercontent.com/304265/77130203-c50d1000-6a35-11ea-84e5-38dfc145b611.png)